modules/config/manager.go

```diff
@@ -1,26 +1,99 @@
 package config
 
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/CodeCollaborate/Server/utils"
+	log "github.com/Sirupsen/logrus"
+	"github.com/kr/pretty"
+)
+
 /**
  * Configuration for the CodeCollaborate Server.
  */
 
 var config *Config
 var configDir = "./config"
 
+func init() {
+	log.SetFormatter(&log.JSONFormatter{})
```

Comment:
They're both just as hard to read, but the JSON formatter allows us to use a structured log reader in future. I didn't see any tradeoff, so it seemed logical